test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala

```diff
@@ -3186,13 +3380,31 @@ class MultipleDisposalsTriageControllerSpec
           if (expectReturnToSummaryLink) messageFromMessageKey("returns.return-to-summary-link")
           else ""
         )
+        expectedAdditionalIdKeyValues.map(a => doc.select(a.id).html() should include(a.value))
+        expectedAdditionalNameAttributeKeyValues.map(v =>
+          doc.select(s"${v.tagName}").attr(v.attributeName, v.attributeValue).text() should include(v.value)
```

Comment:
`s"{v.tagName2}"` is equivalent to just `v.tagName`

Reply:
`shouldBe` would be a better test here as well - one of the strengths of using Jsoup in our unit tests is that we can be much more precise about our testing

Reply:
Done.

Reply:
Just the "includes" above is left as sometimes i take the first element that has an id in the body and search inside it the text. I mean I can try to be a bit more specific but then it will take a just bit more time.